modules/nf-core/samblaster/tests/main.nf.test

```diff
@@ -0,0 +1,57 @@
+nextflow_process {
+
+    name "Test Process SAMBLASTER"
+    script "../main.nf"
+    process "SAMBLASTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "samblaster"
+
+    test("homo_sapiens-test_paired_end_umi_unsorted_bam") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['homo_sapiens']['illumina']['test_paired_end_umi_unsorted_bam'], checkIfExists: true)
```

Comment:
As far as I understood from @maxulysse, `params.modules_testdata_base_path` is now preferred instead of `params.test_data`:

```suggestion
                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina//bam/umi/test.paired_end.umi_unsorted.bam', checkIfExists: true)```

Reply:
Thank you @asp8200